src/components/ex-navigation/ExNavigation.scss

```diff
@@ -0,0 +1,118 @@
+.navigation {
+  width: 20% !important;
+
+  @screen md {
+    width: unset !important;
+  }
+
+  .ex-navigation {
+    &__list {
+      &--hottopicitems,
+      &--topicitems {
+        @apply hidden;
+      }
+
+      &--navigationitems {
+        @apply bg-primary absolute left-0 z-10 flex-wrap hidden w-full px-5 py-3 m-0 mt-3;
+
+        &.open {
+          @apply inline-flex;
+        }
+
+        @screen md {
+          @apply relative flex justify-start p-0 ml-0 bg-transparent;
+        }
+      }
+    }
+
+    &__item {
+      @apply flex w-full pb-2 mb-2 border-b border-white border-solid;
+
+      &:last-child {
+        @apply pb-0 mb-0 border-b-0;
+      }
+
+      @screen md {
+        @apply border-0;
+
+        width: unset;
+      }
+    }
+
+    &__icon {
+      @apply hidden;
+    }
+
+    &__link {
+      @apply text-default text-white no-underline;
+
+      @screen md {
+        @apply text-s bg-primary inline-block px-2 pt-1 pb-0 mr-8 text-white uppercase list-none transition-all duration-300 ease-in-out;
+
+        &:hover {
+          @apply bg-interactive-state-hover-button;
+        }
+      }
+    }
+  }
+
+  .ex-menu-burger {
```

Comment:
The burger menu item have no mouse-over effect. You could use the same background color change like the regular navigation. Should work.

![image](https://user-images.githubusercontent.com/88790097/134134476-d8d9400c-5af6-4bdc-a219-b5b6927d34fa.png)

Reply:
Added hover effect

Reply:
Now there is a double hover effect :D

![image](https://user-images.githubusercontent.com/88790097/134165807-d8574683-0b38-4a0b-ac54-4910eb760d2a.png)

Reply:
My bad. I removed the wrong hover effect